scene/gui/graph_edit.cpp

```diff
@@ -207,6 +192,28 @@ void GraphEditMinimap::_adjust_graph_scroll(const Vector2 &p_offset) {
 	ge->set_scroll_offset(p_offset + graph_offset - camera_size / 2);
 }
 
+void GraphEditMinimap::_bind_methods() {
+	BIND_THEME_ITEM(Theme::DATA_TYPE_STYLEBOX, GraphEditMinimap, panel);
+	BIND_THEME_ITEM_CUSTOM(Theme::DATA_TYPE_STYLEBOX, GraphEditMinimap, node_style, "node");
+	BIND_THEME_ITEM_CUSTOM(Theme::DATA_TYPE_STYLEBOX, GraphEditMinimap, camera_style, "camera");
+	BIND_THEME_ITEM(Theme::DATA_TYPE_ICON, GraphEditMinimap, resizer);
+	BIND_THEME_ITEM(Theme::DATA_TYPE_COLOR, GraphEditMinimap, resizer_color);
+}
+
+GraphEditMinimap::GraphEditMinimap(GraphEdit *p_edit) {
+	ge = p_edit;
+
+	graph_proportions = Vector2(1, 1);
+	graph_padding = Vector2(0, 0);
+	camera_position = Vector2(100, 50);
+	camera_size = Vector2(200, 200);
+	minimap_padding = Vector2(MINIMAP_PADDING, MINIMAP_PADDING);
+	minimap_offset = minimap_padding + _convert_from_graph_position(graph_padding);
+
+	is_pressing = false;
+	is_resizing = false;
```

Comment:
You could move this to the header.

Reply:
I probably should, but I just moved this method around to follow a more consistent order :P

Reply:
I'll keep `minimap_padding` and `minimap_offset`. The latter is calculated and the former is based on a constant which is defined in this file. I could move the constant to the header, but there are other related constants declared together with it, so IMO it's better to keep them all together.